src/lib.rs

```diff
@@ -74,42 +84,93 @@ pub trait InitWith<T> {
     /// assert_eq!(src, dest);
     /// ```
     fn init_with<F>(init: F) -> Self
-        where F: FnMut() -> T,
-              Self: Sized;
+    where
+        F: FnMut() -> T,
+        Self: Sized;
+
+    /// Create a new instance of this type to fill elements
+    /// by mapping the given function over the new array's indices.
+    ///
+    /// # Examples
+    ///
+    /// Prefilling an array of eeven numbers, with no unsafe code
```

Comment:
"eeven"

also would like a colon at the end, to match the other example

Reply:
Fixed